po/ru.po

```diff
@@ -62,32 +62,36 @@ msgstr "Только проценты"
 msgid "Character only"
 msgstr "Только персонаж"
 
-#: src/resources/ui/preferences.ui:51
+#: src/resources/ui/preferences.ui:49
+msgid "Reverse speed"
```

Comment:
Maybe it would be clearer to call this setting `Invert running speed`'?
I'm not a native speaker, but doesn't `Reverse speed` sound like the cat would run backwards?

```po
msgid "Invert running speed"
msgstr "Инвертировать скорость"
```